docs/DatasetEvaluation.md

```diff
@@ -0,0 +1,263 @@
+- `config`: Estimator config YAML file. You can use the default value which points to a YAML file packaged with our docker images.
+- `tb_log_dir`: Path to store tensorboard logs used to visualize the training progress.
+- `checkpoint_dir`: Path to store output Estimator checkpoints. You can use one of the checkpoints for estimator evaluation.
+- `volume_size`: Size of the Kubernetes Persistent Volume Claims (PVC) that will be used to store the dataset. You should change this value according to the dataset that was generated. If you use default settings from [these instructions](RunningSynthDetCloud.md), you should expect `1.2TiB` storage required for 400k images.
+
+> You'll want to change `tb_log_dir`, `checkpoint_dir` to point to a location that is convenient for you and your Kubernetes cluster have permissions to write to. This is typically a GCS path under the same GCP project. Note that an invalid location will cause the job to fail, whereas a path to the local filesystem may run but will be hard to monitor as you won't have easy access to these files inside a docker container.
+
+![pipeline graph](images/kubeflow/pipeline_graph.png)
+
+To open tensorboard for training visualization, you can run the following command:
+
+```bash
+docker run \
+  -p 6006:6006 \
+  -e GOOGLE_APPLICATION_CREDENTIALS=/tmp/key.json \
+  -v $GOOGLE_APPLICATION_CREDENTIALS:/tmp/key.json:ro \
+  -t tensorflow/tensorflow \
+  tensorboard \
+  --host=0.0.0.0 \
+  --logdir=gs://<tb_log_dir>
+```
+
+This assumes you have an environment variable `GOOGLE_APPLICATION_CREDENTIALS` in the host machine that [points](https://cloud.google.com/docs/authentication/production#providing_credentials_to_your_application) to a GCP service account. This service account should have permissions to read `tb_log_dir` to download tensorboard files.
+
+Next, follow the [instructions](#part-3-evaluate-a-model) to evaluate evaluate the performance of this model by running one more pipeline we have prepared. You'll need the location of your model in the next step.
+
+## Part 3: Evaluate a model
+
+In this section, we'll a trained model to generate predictions on a held out test set of real images and measure its performance using well-known performance metrics like [mAP](https://datasetinsights.readthedocs.io/en/latest/datasetinsights.evaluation_metrics.html#datasetinsights.evaluation_metrics.average_precision_2d.MeanAveragePrecisionAverageOverIOU) and [mAR](https://datasetinsights.readthedocs.io/en/latest/datasetinsights.evaluation_metrics.html#datasetinsights.evaluation_metrics.average_recall_2d.MeanAverageRecallAverageOverIOU). We have prepared another kubeflow [pipeline](https://raw.githubusercontent.com/Unity-Technologies/datasetinsights/master/kubeflow/compiled/evaluate_the_model.yaml) for this.
```

Comment:
Opening sentence could be better: specifically this part: "we'll a trained model"

Reply:
Changed to:
>In this section, we'll use a trained model to generate predictions on the test split of UnityGroceries-RealWorld dataset and measure its performance using well-known performance metrics like